Helper/Order.php

```diff
@@ -857,21 +856,20 @@ public function processNewOrder($quote, $transaction)
      */
     protected function hasSamePrice($order, $transaction)
     {
+
+        $this->cartHelper->getRoundAmount($order->getTaxAmount());
         /** @var OrderModel $order */
-        $subtotal = (int) ($order->getSubtotal() * 100);
-        $taxAmount = (int) ($order->getTaxAmount() * 100);
-        $shippingAmount = (int) ($order->getShippingAmount() * 100);
-        $totalAmount = (int) ($order->getGrandTotal() * 100);
+        $taxAmount = $this->cartHelper->getRoundAmount($order->getTaxAmount());
+        $shippingAmount = $this->cartHelper->getRoundAmount($order->getShippingAmount());
+        $grandTotalAmount = $this->cartHelper->getRoundAmount($order->getGrandTotal());
 
-        $transactionSubtotalAmount = $transaction->order->cart->subtotal_amount->amount;
         $transactionTaxAmount = $transaction->order->cart->tax_amount->amount;
         $transactionShippingAmount = $transaction->order->cart->shipping_amount->amount;
         $transactionGrandTotalAmount = $transaction->order->cart->total_amount->amount;
 
-        return abs($subtotal - $transactionSubtotalAmount) <= self::MISMATCH_TOLERANCE
```

Comment:
The subtotal check was removed for 2 reasons:
1. there's a grand totals check, so it is redundant
2. in tests I found $transaction->order->cart->subtotal_amount was not set (null) in the transaction data fetched from Bolt. This should probably be reported <- @matt-thomason